lib/java/logging-service-common/src/main/java/org/enso/logging/service/common/RemoteAppender.java

```diff
@@ -1,4 +1,4 @@
-package org.enso.logging.service;
+package org.enso.logging.service.common;
```

Comment:
It'd be simpler to review these changes without 57cf39b511472c98dadf350beb2ec3691afbfd15, but the review UI doesn't seem to be able to exclude a commit.

Reply:
Unfortunately the https://github.com/enso-org/enso/commit/57cf39b511472c98dadf350beb2ec3691afbfd15 was needed to make the modules work - there was a clash in package names, resulting in JVM boot error.